### ISSUE TITLE
Maintained?

### ISSUE THREAD

**Issue by MicFin** [randym/acts_as_xlsx#13]
I am looking into using this gem but noticed https://github.com/randym/axlsx was updated within the last month and this has not been updated in 2 years.  Is this still the correct gem for a Rails 4 developer to use?

**Comment by decasia** [comment]
@MicFin  agreed that it doesn't seem to be maintained currently, but just for one data point, I'm using it on Rails 4 apps in production without any issues. I would definitely endorse testing it out.

**Comment by jmuheim** [comment]
Yes it would be nice to have this gem updated.

**Comment by n-rodriguez** [comment]
Hi there! Any news?

**Comment by JasonBarnabe** [comment]
Looks like this has been forked into https://github.com/caxlsx/
